### DISCUSSION TITLE
Fetch_many error

### DISCUSSION THREAD

**Discussion by timlee30** [PrefectHQ/prefect#15601] [Q&A]
Hi, 

fetch_many worked fine two weeks before, but recently This is the error. Is there anyone know how to fix this ? Lots of thanks !

 
 File "C:\Python310\lib\site-packages\sqlalchemy\engine\cursor.py", line 1122, in fetchmany
    l = dbapi_cursor.fetchmany(size)
sqlalchemy.exc.DBAPIError: (pyodbc.Error) ('HY010', '[HY010] [Microsoft][ODBC Driver 17 for SQL Server]Function sequence error (0) (SQLFetch)')
(Background on this error at: https://sqlalche.me/e/20/dbapi)

![fetchmany error ](https://github.com/user-attachments/assets/a942ba88-2352-4d19-b49a-2de87c2cbb0c)

**Comment by zzstoatzz** [comment]
hi @timlee30 - thank you for the question, can you elaborate on how you ended up here? the above isn't quite enough information to know how to reproduce
